api/user/user.go

```diff
@@ -11,6 +11,7 @@ type User struct {
 	Email      string
 	HashedPass string
 	KeyID      string
+	Projects   []string
```

Comment:
make a map of project name to user privilege in that proj (as the token will be signed with that)

Reply:
I don't see the point of storing that in 2 places

Reply:
just populating the token with claims we care about, but i guess we always check from db anyways

Reply:
if we put it in the token, we could have a project privs middleware, but yeah not much benefit

Reply:
yeah I think we just introduce more error if we have to update it in 2 places every time